internal/testutil/testutil.go

```diff
@@ -89,11 +90,22 @@ func testContext() (Context, error) {
 		return tc, noProjectID
 	}
 
-	pkgs, err := packages.Load(&packages.Config{Mode: packages.NeedName}, "github.com/GoogleCloudPlatform/golang-samples")
+	cfg := &packages.Config{
+		Mode:  packages.NeedName | packages.NeedFiles,
+		Tests: true,
+	}
+	pkgs, err := packages.Load(cfg, "github.com/GoogleCloudPlatform/golang-samples")
 	if err != nil {
-		return tc, fmt.Errorf("Could not find golang-samples: %v", err)
+		return tc, fmt.Errorf("could not find golang-samples: %v", err)
+	}
+	for _, pkg := range pkgs {
```

Comment:
Given we expect `pkgs` to always be a slice of one package, is the loop best practice vs. just checking the first element? Or perhaps I've misunderstood what `packages.Load` might do?

Reply:
Added a comment. It returns multiple packages (`foo`, `foo.test`, and another `foo`?)